<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IFDRR3S" failures="2" tests="6" name="Test3" time="27.872" errors="0" timestamp="2021-08-31T23:52:55 IST">
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase2" time="0.002"/>
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase2" time="0.002"/>
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase3" time="0.001"/>
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase4" time="11.352">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.Sample_Prep.PreparationCodes.SampleTestNG.testcase4(SampleTestNG.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase4" time="13.224">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.Sample_Prep.PreparationCodes.SampleTestNG.testcase4(SampleTestNG.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase classname="com.Sample_Prep.PreparationCodes.SampleTestNG" name="testcase1" time="0.907"/>
</testsuite> <!-- Test3 -->
